docs/Examples/Go/Component/http-client/index.md

```diff
@@ -0,0 +1,85 @@
+# Go HTTP Client
+
+This repository contains a WebAssembly Component compiled using [TinyGo][tinygo], which:
```

Comment:
It seems like we should change the language so that it doesn't reference "this repository" and maybe swap in the `wash new ...` command to pull it down?

Reply:
I was going to add a page after this merges to the top level that explains what the docs are here with links to each of the repositories

Reply:
So after talking to @ericgregory about it, we should change the wording. I think the plan is we get this merged so _something_ is on the site and I'll work on a quick follow up today to change the wording